### PR TITLE
Update Go version to 1.19

### PR DESCRIPTION
**Pull request by 4lexvav**

### CODE REVIEW

File: .github/workflows/validate-merge.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16.4
+        go-version: 1.19.1
```
Comment (SyntaxNode) [inline .github/workflows/validate-merge.yml]: Please keep internally consistent. If the dockerfil is using 19.2, let' use 19.2 here too.

File: adapters/adapterstest/test_json.go [hunk 2]
```diff
@@ -22,38 +22,37 @@ import (
 // RunJSONBidderTest is a helper method intended to unit test Bidders' adapters.
 // It requires that:
 //
-//   1. Bidders communicate with external servers over HTTP.
-//   2. The HTTP request bodies are legal JSON.
+//  1. Bidders communicate with external servers over HTTP.
+//  2. The HTTP request bodies are legal JSON.
```
Comment (SyntaxNode) [inline adapters/adapterstest/test_json.go]: To other reviewers, the following comment changes are autogenerated. Go 1.19 auto formats comments while earlier versions of Go do not.

**Review by SyntaxNode — COMMENTED** [review]
Looks good. Please open this as a PR to the main repository.

**Comment by 4lexvav** [comment]
Closing this PR as we've opened one to the main repository: https://github.com/prebid/prebid-server/pull/2428

**Pull request closed without merging**